src/fides/api/main.py

```diff
@@ -322,6 +325,48 @@ def read_other_paths(request: Request) -> Response:
     return get_admin_index_as_response()
 
 
+# def create_or_update_parent_user(db: Session) -> None:
```

Comment:
is this intended? feels like it may have been a temporary change you made locally that wasn't meant to be part of the commit....or i'm missing something 😄

Reply:
I decided to move it to make `main.py` cleaner and forgot to remove this 🤦